dashboards/flows-by-science-discipline.json

```diff
@@ -489,7 +489,7 @@
               "type": "sum"
             }
           ],
-          "query": "meta.sensor_id.keyword:$Sensors && (meta.scireg.src.discipline.keyword:$discipline OR meta.scireg.dst.discipline.keyword:$discipline)",
+          "query": "meta.sensor_id.keyword:$Sensors && (meta.scireg.src.discipline.keyword:$discipline or meta.scireg.dst.discipline.keyword:$discipline)",
```

Comment:
This is definitely a regression. I think you might have made this starting from the wrong place? I confirmed that the 1.3.0 branch already has the or -> OR fix in it. Maybe need to redo this.

Reply:
Ah, i'll look into it. Not sure how that happened